src/main/scala/cromwell/engine/callactor/CallActor.scala

```diff
@@ -196,7 +196,7 @@ trait CallActor extends LoggingFSM[CallActorState, CallActorData] with CromwellA
     val message = executionResult match {
       case SuccessfulBackendCallExecution(outputs, executionEvents, returnCode, hash, resultsClonedFrom) =>
         WorkflowActor.CallCompleted(key, outputs, executionEvents, returnCode, if (workflowDescriptor.writeToCache) Option(hash) else None, resultsClonedFrom)
-      case SuccessfulFinalCallExecution => WorkflowActor.CallCompleted(key, Map.empty, Seq.empty, 0, None, None)
+      case SuccessfulFinalCallExecution => WorkflowActor.CallCompleted(key, Map.empty, Seq.empty, ScriptReturnCode(0), None, None)
```

Comment:
This is for the `SuccessfulFinalCallExecution` which doesn't really have a return code since it's not even a script (for now at least). Problem is returnCode is mandatory in a bunch of places where it doesn't really make sense. IMO this is unrelated to the ticket and would be a non-trivial refactor I think.